### PR TITLE
Removes abilities from admins.

### PR DESCRIPTION
**Pull request by ashercodeorg**
Also adds the `Game` ability to users with `UserPermission::LEVELBUILDER`.

### CODE REVIEW

**Comment by ashercodeorg** [comment]
Thanks for your help @aoby with that last commit!

File: dashboard/app/models/ability.rb [hunk 1]
```diff
@@ -171,9 +171,10 @@ def initialize(user)
     # permissions.
     if user.persisted? && user.permission?(UserPermission::LEVELBUILDER)
       can :manage, [
+        Game,
         Level,
         Script,
-        ScriptLevel
+        ScriptLevel,
```
Comment (aoby) [inline dashboard/app/models/ability.rb]: nit: trailing `,` not needed

File: dashboard/test/controllers/script_levels_controller_test.rb [hunk 2]
```diff
@@ -962,12 +962,12 @@ def create_admin_script
     assert_response :forbidden
   end
 
-  test "should get show of admin script if signed in as admin" do
+  test "should not get show of admin script if signed in as admin" do
```
Comment (aoby) [inline dashboard/test/controllers/script_levels_controller_test.rb:968]: Who should be able to see the admin script, if not admin? This is a bit confusing. Is it meant to be levelbuilder? Maybe rename levelbuilder_script? Or At least add a comment in the test.

File: dashboard/app/models/ability.rb (hunk 1 above)
Comment (ashercodeorg) [inline dashboard/app/models/ability.rb]: Done (remnant of debugging).

File: dashboard/test/controllers/script_levels_controller_test.rb (hunk 2 above)
Comment (ashercodeorg) [inline dashboard/test/controllers/script_levels_controller_test.rb:968]: No one. :) The `admin_only` and `student_of_admin` scripts are being deprecated and removed (as part of this work). Adding a TODO to remove (consolidate) this (these) test(s), though for now, I'm keeping the test as it is "useful".

**Comment by aoby** [comment]
Please give a little more detail about the deferred work in that TODO comment.

Otherwise, LGTM

**Pull request merged**